src/Microsoft.AspNet.Razor.Runtime/Encodings/NullHtmlEncoder.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace System.Text.Encodings.Web
+{
+    /// <summary>
+    /// A <see cref="HtmlEncoder"/> that does not encode. Should not be used when writing directly to a response
+    /// expected to contain valid HTML.
+    /// </summary>
+    public class NullHtmlEncoder : HtmlEncoder
```

Comment:
An encoder that doesn't encode would be more general use imo since we're now all using `IHtmlContent` whos only signature expects an encoder.